src/Microsoft.AspNetCore.Mvc.Razor/RazorViewEngineOptions.cs

```diff
@@ -120,39 +143,5 @@ public Action<RoslynCompilationContext> CompilationCallback
                 _compilationCallback = value;
             }
         }
-
```

Comment:
` IList<MetadataReference> AdditionalCompilationReferences` and `Action<RoslynCompilationContext> CompilationCallback` are both still on this type. So we're not necessarily doing much about

> Putting a Roslyn type like this in DI via options causes a LOT of things to be jitted even when we don't compile anything